docs/onboarding.md

```diff
@@ -43,3 +43,18 @@ Then add your SSH key _also_ as the commit signing key:
 1. Open [New SSH Key](https://github.com/settings/ssh/new) page.
 2. Select `Key type`: `Signing key`.
 3. Copy the content of `~/.ssh/id_rsa.pub` file (or other file you used in `user.signingKey` above).
+
+If you're on WSL2, this may help:
+
+- Add those lines to `~/.gnupg/gpg.conf`
```

Comment:
Are these steps `do 1 and do 2` or it is `do 1 or do 2`?

If it is `and` steps, it is better to use `<ol>` (`1. `) list here.